docs/compute.md

```diff
@@ -309,6 +309,24 @@ With this simple but first initial algorithm, we managed to mitigate automatical
 
 You have more than 2 hosts? Again, same idea, it will also work.
 
+## VM anti-affinity
+
+VM anti-affinity is a feature that prevents VMs with the same user tags to run on the same host. This functionality is available directly in the load-balancer plugin.
```

Comment:
At this stage I don't know what the load-balancer plugin is. I suggest to write "in Xen Orchestra's load-balancer plugin"

Reply:
I think we should have a dedicated load balancer section, but the frontier between XO and XCP-ng doc is thin on that matter.

But anyway, I think this all section should be added to XO docs and THEN we'll see what to do with the feature in XCP-ng doc.

Reply:
There is already a section for that, but this feature is not a load-balancer mode and it's not a load-balancing element. It would be interesting to change the plugin name before updating the doc. Like we said in the past ` Resource Scheduler` or a similar name to group anti-affinity functionality and load-balancer plans.

Reply:
I think we still should update Xen Orchestra doc in the load balancer section. We'll rename the section later, but anti-affinity is related to the load balancing plugin so far. That's where I intended to put it first.